### PR TITLE
CLEANING: Typing & Misc

### PR DESCRIPTION
**Pull request by oualib**
- Cleaning wrong init.
 - startswith and endswith instead of ugly equalities.

TODO:
 - except 
 - warning and errors

=> end of code cleaning

### CODE REVIEW

File: verticapy/core/vdataframe/_sys.py [hunk 1]
```diff
@@ -416,56 +416,45 @@ def expected_store_usage(self, unit: str = "b") -> TableSample:
...
-            elif ctype[0:7] == "varchar":
+            elif ctype.startswith("varchar"):
                 try:
                     size = int(ctype.split("(")[1].split(")")[0])
-                except:
+                except IndexError:
                     size = 80
                 maxsize, expsize = size, self[column].store_usage()
-            elif (ctype[0:4] == "char") or (ctype[0:3] == "geo") or ("binary" in ctype):
+            elif ctype.startswith("geo") or ctype.endswith(
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_sys.py:444]: You may want to double check this. Initially you were checking for [0:4] "char".
And initially there is no endswith as well.

File: verticapy/core/vdataframe/_sys.py [hunk 2]
```diff
@@ -416,56 +416,45 @@ def expected_store_usage(self, unit: str = "b") -> TableSample:
...
                 try:
                     size = int(ctype.split("(")[1].split(")")[0])
                     maxsize, expsize = size, size
-                except:
-                    if ctype[0:3] == "geo":
+                except IndexError:
+                    if ctype.startswith("geo"):
                         maxsize, expsize = 10000000, 10000
-                    elif "long" in ctype:
+                    elif ctype.startswith("long"):
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_sys.py:453]: IN is exchanged with STARTSWITH

**Review by mail4umar — CHANGES_REQUESTED** [review]

File: verticapy/core/vdataframe/_sys.py (hunk 1 above)
Comment (oualib) [reply verticapy/core/vdataframe/_sys.py:444]: no it is what I want to check

File: verticapy/core/vdataframe/_sys.py (hunk 2 above)
Comment (oualib) [reply verticapy/core/vdataframe/_sys.py:453]: yes the new code is correct.

**Review by mail4umar — APPROVED** [review]

**Pull request merged**